### ISSUE TITLE
ion-selects don't show in an ion-modal (for e.g., changing type)

### ISSUE THREAD

**Issue by gbj** [gbj/venite#35]

**Comment by gbj** [comment]
See Ionic issue here https://github.com/ionic-team/ionic/issues/21181

See solution here https://forum.ionicframework.com/t/on-select-is-hidden-behind-ion-modal/189703/3

**Comment by gbj** [comment]
```
import { Component, h } from '@stencil/core';
import { modalController } from '@ionic/core';
@Component({
  tag: 'app-home',
  styleUrl: 'app-home.css',
})
export class AppHome {
  async openModal() {
    const modal = await modalController.create({
      component: 'app-modal',
    });
    modal.onWillDismiss().then((x) => {
      console.log(x);
    });

    await modal.present();
  }
```

**Comment by gbj** [comment]
Fixed in 365f214858450ee68bc243ca31a68b1a04baafd
